interface/patient_file/encounter/view_form.php

```diff
@@ -6,14 +6,17 @@
 
 include_once("../../globals.php");
 
-if (substr($_GET["formname"], 0, 3) === 'LBF') {
+$clean_formname=filter_var($_GET["formname"],FILTER_SANITIZE_STRING);
```

Comment:
Why do you need to do this?
This script will die at check_file_dir_name() function call if has anything without following charaxcter: A-Za-z0-9_.-

Reply:
This is actually a good example, where a change is made that can potentially cause negative impact and not close the vulnerabilties. If this is an LBF form, the the raw GET formname goes to this script https://github.com/openemr/openemr/blob/master/interface/forms/LBF/new.php
(search formname to see all the places that need securing, but not in a one size fits all fashion)